qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -1068,6 +1068,195 @@ def GaussianSquareDrag(
     return instance
 
 
+def GaussianSquareEcho(
+    duration: Union[int, ParameterExpression],
+    amp: Union[float, ParameterExpression],
+    sigma: Union[float, ParameterExpression],
+    width: Optional[Union[float, ParameterExpression]] = None,
+    angle: Optional[Union[float, ParameterExpression]] = 0.0,
+    active_amp: Optional[Union[float, ParameterExpression]] = 0.0,
+    active_angle: Optional[Union[float, ParameterExpression]] = 0.0,
+    risefall_sigma_ratio: Optional[Union[float, ParameterExpression]] = None,
+    name: Optional[str] = None,
+    limit_amplitude: Optional[bool] = None,
+) -> SymbolicPulse:
+    """An echoed Gaussian square pulse with an active tone overlaid on it.
+    
+    Exactly one of the ``risefall_sigma_ratio`` and ``width`` parameters has to be specified.
+    If ``risefall_sigma_ratio`` is not ``None`` and ``width`` is ``None``:
+    .. math::
+        \\text{risefall} &= \\text{risefall_sigma_ratio} \\times \\text{sigma}\\\\
+        \\text{width} &= \\text{duration} - 2 \\times \\text{risefall}
+    If ``width`` is not None and ``risefall_sigma_ratio`` is None:
+    .. math:: \\text{risefall} = \\frac{\\text{duration} - \\text{width}}{2}
+    The Gaussian Square Echo pulse is composed of three pulses. First, a Gaussian Square pulse
+    :math: `f_{echo}(x)` with amplitude ``amp`` and phase ``angle`` playing for half duration,
+    followed by a second Gaussian Square pulse :math: `-f_{echo}(x)` with opposite amplitude
+    and same phase playing for the rest of the duration. Third a Gaussian Square pulse
+     :math: `f_{active}(x)` with amplitude ``active_amp`` and phase ``active_angle``
+    playing for the entire duration. The Gaussian Square Echo pulse :math: `g_e()`
+    can be written as:
+    .. math::
+
+        g_e(x) &= \\begin{cases}\
+            f_{\\text{active}} + f_{\\text{echo}}(x)\
+                & x < \\frac{\\text{duration}}{2}\\\\
+            f_{\\text{active}} - f_{\\text{echo}}(x)\
+                & \\frac{\\text{duration}}{2} < x\\\\
+
+
+
+    References:
+        1. |citation1|_
+
+        .. _citation1: https://journals.aps.org/prxquantum/abstract/10.1103/PRXQuantum.1.020318
```

Comment:
I don't think this paper needs to be cited. I would cite https://arxiv.org/abs/2008.08571 and maybe refer specifically to the Direct CX Gate section. That paper covers the gaussian square echo pulse and refers to this earlier reference for rotary echoes, but this earlier paper only uses rotary echoes with the ECR gate.